alacritty_terminal/src/term/cell.rs

```diff
@@ -144,14 +377,16 @@ impl GridCell for Cell {
 
     #[inline]
     fn reset(&mut self, template: &Self) {
-        *self = Cell { bg: template.bg, ..Cell::default() };
+        *self = template.colors.bg().into();
     }
 }
 
 impl From<Color> for Cell {
     #[inline]
     fn from(color: Color) -> Self {
-        Self { bg: color, ..Cell::default() }
+        let mut cell = Cell::default();
+        cell.colors.set_bg(color);
+        cell
```

Comment:
I'd be somewhat worried of the optimization of this, it's probably a bit hot iirc. So changing this to something that is significantly less efficient will probably affect performance at least a little bit?